site/docs/Learn-Schema.md

```diff
@@ -226,17 +231,17 @@ For example, here are some types that might implement `Character`:
 ```graphql
 type Human implements Character {
   id: String!
-  name: String
+  name: String!
```

Comment:
On the other hand, in some typed languages having everything be nullable makes it a bit annoying to work with the data. I talked to Dan and he also had a few ideas about it which basically came down to - data that involves fetches should always be nullable in case the fetch fails, but scalar fields on objects that are returned as one unit should maybe be non-null.

But that's mostly an academic concern and I agree too many `!` clutter the schema.